addons/binding/org.openhab.binding.netatmo/src/main/java/org/openhab/binding/netatmo/handler/welcome/NAWelcomeHomeHandler.java

```diff
@@ -0,0 +1,249 @@
+/**
+ * Copyright (c) 2014-2016 by the respective copyright holders.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.openhab.binding.netatmo.handler.welcome;
+
+import static org.openhab.binding.netatmo.NetatmoBindingConstants.*;
+
+import java.util.Collections;
+import java.util.Comparator;
+import java.util.HashMap;
+import java.util.concurrent.ScheduledFuture;
+import java.util.concurrent.TimeUnit;
+
+import org.eclipse.smarthome.core.library.types.DecimalType;
+import org.eclipse.smarthome.core.library.types.OnOffType;
+import org.eclipse.smarthome.core.library.types.StringType;
+import org.eclipse.smarthome.core.thing.Thing;
+import org.eclipse.smarthome.core.thing.ThingStatus;
+import org.eclipse.smarthome.core.thing.ThingStatusDetail;
+import org.eclipse.smarthome.core.thing.binding.ThingHandler;
+import org.eclipse.smarthome.core.types.State;
+import org.eclipse.smarthome.core.types.UnDefType;
+import org.openhab.binding.netatmo.config.NetatmoWelcomeHomeConfiguration;
+import org.openhab.binding.netatmo.handler.NetatmoWelcomeHandler;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import io.swagger.client.model.NAWelcomeEvents;
+import io.swagger.client.model.NAWelcomeHomeData;
+import io.swagger.client.model.NAWelcomeHomes;
+import io.swagger.client.model.NAWelcomePersons;
+
+/**
+ * {@link NAWelcomeHomeHandler} is the class used to handle the Welcome Home Data
+ *
+ * @author Ing. Peter Weiss - Welcome camera implementation
+ *
+ */
+
+public class NAWelcomeHomeHandler extends NetatmoWelcomeHandler<NetatmoWelcomeHomeConfiguration> {
+    private static Logger logger = LoggerFactory.getLogger(NAWelcomeHomeHandler.class);
+
+    private ScheduledFuture<?> bridgeHandlerJob;
+    private ScheduledFuture<?> refreshJob;
+
+    private int iPerson = -1;
+    private int iUnknown = -1;
+    private boolean bSomebodyAtHome = false;
+
+    public NAWelcomeHomeHandler(Thing thing) {
+        super(thing, NetatmoWelcomeHomeConfiguration.class);
+    }
+
+    @Override
+    public void initialize() {
+        super.initialize();
+        initBridgeScheduler();
+    }
+
+    private void initBridgeScheduler() {
+        logger.debug("scheduling bridge thread to run every {} ms", 60000);
+        bridgeHandlerJob = scheduler.scheduleWithFixedDelay(new Runnable() {
```

Comment:
I don't understand the need to schedule that every minute.
You have to urn this code once in initialize() and you have another method trigerred when the bridge changes its state and you have to put the same code in this method.